packages/react-native/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-macos",
-  "version": "0.76.0",
+  "version": "0.76.0-ready",
```

Comment:
:heart:

Reply:
The last time I tried this it didn't work... but I shall try :D